installers/flash-tarball/radiasoft-download.sh

```diff
@@ -62,36 +62,21 @@ flash_tarball_patch_and_update_tgz() {
 > F90FLAGS = -fallow-argument-mismatch
 EOF
 
-    # The following patches both contain trailing whitespace because
-    # the original source file contains it. Keeping the trailing
-    # whitespace makes the patch smaller which makes the important
-    # change in the patch easier to see.
-    patch --quiet bin/setup.py <<'EOF'
-@@ -13,7 +13,7 @@
- 
- ########################### START SCRIPT ################################
- def main():
--    sys.setcheckinterval(10000) # this is not threaded application
-+    sys.setswitchinterval(10000) # this is not threaded application
- 
-     parseCmd.init()
-     # setup.py is FLASH_HOME/bin/setup.py
-EOF
     patch --quiet tools/python/flmake/setup.py <<'EOF'
 @@ -32,7 +32,7 @@
      global RUNTIME_FILES
- 
+
      # this is not a threaded application
 -    sys.setcheckinterval(10000)
```

Comment:
Are the two setup.py's different?

Reply:
Yes, for whatever reason tools/python/flmake/setup.py has not been fixed.